content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/150-using-prisma-migrate.mdx

```diff
@@ -52,7 +71,38 @@ This command does two things:
 
 > **Note**: `generate` is called under the hood by default, after running `prisma migrate dev`. If the `prisma-client-js` generator is defined in your schema, this will check if `@prisma/client` is installed and install it if it's missing.
 
-Great, you now created three tables in your database with Prisma Migrate 🚀
+</SwitchTech>
+
+<SwitchTech technologies={['*', 'planetscale']}>
+
+You are now ready to push your new schema to your database. Connect to your `main` branch using the instructions in [Connect your database](https://www.prisma.io/docs/getting-started/setup-prisma/start-from-scratch/relational-databases/connect-your-database-typescript-planetscale/).
+
+Now use the `db push` CLI command to push to the `main` branch:
+
+```terminal
+npx prisma db push
+```
+
+</SwitchTech>
+
+<SwitchTech technologies={['*', 'sqlserver']}>
+
+To map your data model to the database schema, you need to use the `prisma migrate` CLI commands:
+
+```terminal
+npx prisma migrate dev --name init
+```
+
+This command does two things:
+
+1. It creates a new SQL migration file for this migration
+1. It runs the SQL migration file against the database
+
+> **Note**: `generate` is called under the hood by default, after running `prisma migrate dev`. If the `prisma-client-js` generator is defined in your schema, this will check if `@prisma/client` is installed and install it if it's missing.
+
+</SwitchTech>
+
+Great, you now created three tables in the `main` branch with Prisma Migrate 🚀
```

Comment:
This one needs to go into the Techswitcher now as "main branch" only applies to PlanetScale, for all other "Great, you now created three tables in your database with Prisma Migrate 🚀" is still correct.

Alternatively: Do not differentiate and go back to the old wording. Should be fine.

Reply:
Thanks! Went back to old wording.